### DISCUSSION TITLE
Navigation Menu - Remove on hover for NavigationMenuTrigger

### DISCUSSION THREAD

**Discussion by BradleyHennen** [radix-ui/primitives#1381] [Help] [answered]
Is there currently a way to stop the Navigation Menu trigger from opening the menu on hover? 

I know the component is in beta at the time of posting. If there currently isn't a way to disable on hover, is this at all planned for the full release?

**Comment by andy-hook** [comment, marked as answer]
You can disable this behaviour by calling `preventDefault` on the relevant events for `Trigger` and `Content`

https://codesandbox.io/s/unruffled-robinson-duwvgn?file=/App.js

Note that `Trigger` binds to `onPointerMove / onPointerLeave` while `Content` binds to `onPointerEnter / onPointerLeave`
